x/clp/client/cli/tx_test.go

```diff
@@ -1,14 +1,6 @@
 package cli_test
 
-import (
-	"io/ioutil"
-	"testing"
-
-	"github.com/Sifchain/sifnode/x/clp/client/cli"
-	"github.com/cosmos/cosmos-sdk/client/flags"
-	"github.com/spf13/viper"
-	"github.com/stretchr/testify/assert"
-)
+/* TODO: convert to integration tests
```

Comment:
Is this still a TODO?

Reply:
No we already have the same tests under tests/integration/src/py/test_liquidity_pools.py .